R/fiveyeargwsum-styles.R

```diff
@@ -1,12 +1,20 @@
-getFiveyearStyle <- function(data, info=NULL) {
+getFiveyearStyle <- function(data, info=NULL, invertedPlot = FALSE) {
 
     x <- data[[1]]$time
     y <- data[[1]]$value
+    
+    #semantics for min/max are swapped on inverted plots
+    minMaxLabels = c("Max. Instantaneous", "Min. Instantaneous");
+    if(invertedPlot) {
+      minMaxLabels = c("Min. Instantaneous", "Max. Instantaneous");
+    }
+    
```

Comment:
I think we should keep data parsing/munging stuff in the -`data.R` script. Then, you can have L1-2 be:

```
getFiveyearStyle <- function(data, info=NULL, ...) {
     args <- list(...)
```

and then the max/min `legend.name = paste(args$minMaxLabels[1], info$type, ":", y)`